x-pack/plugins/observability/public/components/slo/burn_rate/alert_details/components/alerts_history/alerts_history_panel.tsx

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLink,
+  EuiLoadingChart,
+  EuiLoadingSpinner,
+  EuiPanel,
+  EuiStat,
+  EuiText,
+  EuiTextColor,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useAlertsHistory } from '@kbn/observability-alert-details';
+import rison from '@kbn/rison';
+import { ALERT_RULE_PARAMETERS } from '@kbn/rule-data-utils';
+import { GetSLOResponse } from '@kbn/slo-schema';
+import moment from 'moment';
+import React from 'react';
+import { convertTo } from '../../../../../../../common/utils/formatters';
+import { WindowSchema } from '../../../../../../typings';
+import { useKibana } from '../../../../../../utils/kibana_react';
+import { ErrorRateChart } from '../../../../error_rate_chart';
+import { BurnRateAlert, BurnRateRule } from '../../alert_details_app_section';
+import { getActionGroupFromReason } from '../../utils/alert';
+
+interface Props {
+  slo?: GetSLOResponse;
+  alert: BurnRateAlert;
+  rule: BurnRateRule;
+  isLoading: boolean;
+}
+
+export function AlertsHistoryPanel({ rule, slo, alert, isLoading }: Props) {
+  const {
+    services: { http },
+  } = useKibana();
+  const { isLoading: isAlertsHistoryLoading, data } = useAlertsHistory({
+    featureIds: ['slo'],
+    ruleId: rule.id,
+    dateRange: {
+      from: 'now-30d',
+      to: 'now',
+    },
+    http,
+  });
+
+  const actionGroup = getActionGroupFromReason(alert.reason);
+  const actionGroupWindow = (
+    (alert.fields[ALERT_RULE_PARAMETERS]?.windows ?? []) as WindowSchema[]
+  ).find((window: WindowSchema) => window.actionGroup === actionGroup);
+  const dataTimeRange = {
+    from: moment().subtract(30, 'day').toDate(),
```

Comment:
@kdelemme Is it always the history of the last 30 days? Or depends on the SLO definition time window?

Reply:
it's independent of the SLO time window, it is related to the alerts history, and I guess 30days is what we use in the other alert details page